motr/setup_dix.c

```diff
@@ -344,7 +344,10 @@ M0_INTERNAL int m0_cs_dix_setup(struct m0_motr *cctx)
 	M0_ASSERT(pver != NULL);
 	pm = &pver->pv_mach;
 	m0_rwlock_read_lock(&pm->pm_lock);
-	M0_ASSERT(!pver->pv_is_dirty);
+	/*
+ 	 * Enable this logic once multiple DIX pvers available. 
```

Comment:
[Suggestion]
It will be good to add FIXME/TODO kind of label here as this is AI for future.
same for below comment.

Reply:
Done.